docs/install/docker.md

```diff
@@ -9,8 +9,8 @@ nav_order: 0
 
 The recommended and easiest way to get started is with Docker. To learn more about Docker head on over to their [website](https://www.docker.com/).
 
-```bash
-docker run philosowaffle/peloton-to-garmin:stable -v ./configuration.local.json:/app/configuration.local.json -v ./output:/app/output

```

Comment:
Goes to show what happens when I copy/paste someone elses docker run command. I pretty much exclusively use compose files.

Good to have the docs updated with correct command though.